docs/index.md

```diff
@@ -381,6 +385,8 @@ In addition to the authentication options, the following options are also suppor
 | Name | Description | Default Value |
 |------|-------------|---------------|
 | `telemetry_optout` | Opting out of telemetry will remove the User-Agent and session id headers from the requests made to the Power Platform service.  There is no other telemetry data collected by the provider.  This may affect the ability to identify and troubleshoot issues with the provider. | `false` |
+| `partner_id` | Optional GUID for Customer Usage Attribution. When set, the value is appended to the User-Agent header as `pid-<GUID>`. | |
+| `disable_terraform_partner_id` | When `true`, suppresses the default Terraform partner ID when no custom `partner_id` is provided. | `false` |
```

Comment:
I suppose I don't understand the logic. but is partnet_id is not defined is that not enough to say its disabled? Or there is a reason why we have disable_terraform_partner_id to disable but keep partner_id value?

Reply:
this is a consistency thing with the other providers.  no partner_id still sends a default value.  disabled sends no value at all.  Weird but that's how the other providers do it.